lib/cstruct.mli

```diff
@@ -262,6 +262,9 @@ val blit_to_string: t -> int -> string -> int -> int -> unit
     valid segment of [src], or if [dstoff] and [len] do not designate
     a valid substring of [dst]. *)
 
+val set: t -> int -> unit
```

Comment:
Yes, it's because of consistency with `set_uint8`. I think it would be confusing otherwise, as to what else is different between the two.